sources/packages/backend/apps/db-migrations/src/sql/ApplicationOfferingChangeRequests/Create-application-offering-change-requests.sql

```diff
@@ -0,0 +1,52 @@
+CREATE TABLE sims.application_offering_change_requests (
+  id SERIAL PRIMARY KEY,
+  application_id INT NOT NULL REFERENCES sims.applications(id),
+  change_offering_id INT NOT NULL REFERENCES sims.education_programs_offerings(id),
+  active_offering_id INT NOT NULL REFERENCES sims.education_programs_offerings(id),
+  application_offering_change_request_status sims.application_offering_change_request_status_types NOT NULL,
+  assessed_date TIMESTAMP WITH TIME ZONE,
+  assessed_by INT REFERENCES sims.users(id),
+  student_action_date TIMESTAMP WITH TIME ZONE,
+  reason VARCHAR(500) NOT NULL,
+  ministry_note_id INT REFERENCES sims.notes(id),
+  -- Audit columns
+  created_at TIMESTAMP WITH TIME ZONE NOT NULL DEFAULT NOW (),
+  updated_at TIMESTAMP WITH TIME ZONE NOT NULL DEFAULT NOW (),
+  creator INT NULL DEFAULT NULL REFERENCES sims.users(id) ON DELETE
```

Comment:
I believe that we stooped using the `ON DELETE` also for these ones, right?
I found some examples in the source code no longer using them.

Reply:
mm, I when was doing this I found it on most of the recent ones, but I will remove it